modules/alma/lib/AlmaClient/AlmaClient.class.php

```diff
@@ -814,6 +835,7 @@ public function get_availability($alma_ids) {
       $data[$record->getAttribute('id')] = array(
         'reservable' => ($record->getAttribute('isReservable') == 'true') ? TRUE : FALSE,
         'available' => ($record->getAttribute('isAvailable') == 'yes') ? TRUE : FALSE,
+        'availability_information' => ($record->getAttribute('availabilityInformation') == 'noHolding') ? FALSE : TRUE,
```

Comment:
The `? FALSE : TRUE` is not needed.
The comparison result is already either TRUE or FALSE.